doc/changelog.rst

```diff
@@ -8,6 +8,7 @@ Changelog
 * :feature:`598` Drop support for python version < 3.6
 * :feature:`-` Added support for tagging test parameterization values
 * :feature:`-` Added a plugin which provides links to test logs which have been archived by a CI system
+* :feature:`-` Convert float and integer test details to strings in xunit output files
```

Comment:
I think this is actually a bugfix... phrasing should be that the behavior was corrected for test details containing floats in conjunction with xunit...